rosette/src/Prim.cc

```diff
@@ -145,7 +145,14 @@ DEF("runtime-error", obRuntimeError, 1, MaxArgs) {
         BASE(ARG(i))->printQuotedOn(stderr);
         putc('\n', stderr);
     }
-    return INVALID;
+
+    // NB(leaf): I think it sucks to have a runtime error that can
+    // cause the binary to return 0 to the shell. Aborting here makes
+    // a runtime error into a hard failure, instead of a soft one.
+    // The perror() serves as a reminder to people what the problem
+    // might be.
+    perror("Nota Bene: did you forget to set ulimit -s unlimited?");
+    abort();
 }
```

Comment:
At one point we talked about bringing the ulimit setting internal to Rosette.  Is this possible? Is now the time?

Reply:
I would rather this wort exist as a reminder to make the GC algorithm not overflow a reasonable stack. This isn't that hard, but today is not the day.